monitoring/osd-understanding-the-monitoring-stack.adoc

```diff
@@ -0,0 +1,27 @@
+[id="understanding-the-monitoring-stack"]
+= Understanding the monitoring stack
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: understanding-the-monitoring-stack
+
+toc::[]
+
+User-workload monitoring lets you monitor your own applications using a different Prometheus instance on your own worker nodes, without interfering with Red Hat Site Reliability Engineer (SRE) platform metrics.
```

Comment:
Given that this is the first paragraph in this assembly and book, you could provider a more high-level introduction here. The reader might wonder what is meant by a "different Prometheus instance" because they might not yet have read the details about the architecture which follow.

For example, you could change this to something like the following example:

"In {product-title}, you can monitor your own projects in isolation from Red Hat Site Reliability Engineer (SRE) platform metrics. You can monitor your own projects without the need for an additional monitoring solution."

Reply:
Thanks!